package-lock.json

```diff
@@ -2662,14 +2662,15 @@
       "dev": true
     },
     "bulk-data-utilities": {
-      "version": "git+https://github.com/projecttacoma/bulk-data-utilities.git#92831b38717ee8ef1350d57a06423c09092ed7fa",
+      "version": "git+https://github.com/projecttacoma/bulk-data-utilities.git#0fc2f3dadba664fadeba2b285bcc30362cb3a043",
```

Comment:
Not sure how this happened, but this actually downgrades the bulk-data-utilities dependency (which could be why you're seeing issues with bulk data stuff that others aren't). the `92831b` commit is the proper hash we should be using